crates/core/liveview-native-core-jetpack/core/build.gradle.kts

```diff
@@ -168,11 +168,31 @@ tasks.whenObjectAdded {
 }
 
 tasks.register<Jar>("nativeLibJar") {
-    archiveBaseName.set("liveview-native-core-host")
+    val osName = System.getProperty("os.name")
+    val osArch = System.getProperty("os.arch")
+
+    val targetOs = when {
+        osName == "Mac OS X" -> "darwin"
```

Comment:
This isn't an edit, I had to hand confirm for myself, Yes kotlin does infact describe MacOs 11+ as Mac Os X